### DISCUSSION TITLE
editor.loadData() data parameter schema

### DISCUSSION THREAD

**Discussion by raywong-fb** [GrapesJS/grapesjs#4170] [Q&A] [answered]
I haven't been able to find specific documentation as to what the available object properties are for the `editor.loadData(data)` data param. Specifically, my use case is to be able to switch between static html templates, where some templates have multiple pages, and some are just a single page w/ simple html.

This might be a silly question, but for the templates that aren't multiple pages, is it possible to just load from a raw html string?

**Comment by ebalajier** [comment, marked as answer]
[Try storeData() ](https://grapesjs.com/docs/api/editor.html#storedata)

This should give you the schema structure in the console log.
